cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -42,6 +42,7 @@ const (
 
 	defaultV18AdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
 	defaultV19AdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota"
```

Comment:
I would name this deprecatedV19AdmissionControl

Reply:
@luxas kindly review